demos/custom_prompt_test.py

```
@@ -0,0 +1,240 @@
+                print(f"   🎯 Confidence: {answer.confidence:.1f}")
+        
+        # Show the value of the systematic approach
+        print(f"\n🎯 QADI VALUE DEMONSTRATION:")
+        print("-" * 50)
+        print(f"✅ Systematic Analysis: 4 cognitive phases applied")
+        print(f"✅ Comprehensive Coverage: {len(result.synthesized_ideas)} insights considered") 
+        print(f"✅ Practical Output: {len(result.extracted_answers.direct_answers) if result.extracted_answers else 0} actionable strategies")
+        print(f"✅ Question Understanding: Automatically detected as '{result.extracted_answers.question_type if result.extracted_answers else 'unknown'}'")
+        print(f"✅ Contextual Adaptation: Used bookstore-specific context in analysis")
+        
+        return True
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)